src/modules/header/components/Header.js

```diff
@@ -168,7 +143,7 @@ export class Header extends BaElement {
 				<mask class="header__background">
 				</mask>
```

Comment:
@alsturm Tag looks strange to me...

Reply:
change to div works (don't know why this is so)